### PR TITLE
Restaurant page code review

### PR DESCRIPTION
**Pull request by majovanilla**
Requesting code review for the project.

All requirements completed:

- [x] Use webpack
- [x] Use ES6
- [x] Use src and dist folders to organize code
- [x] Use of JS to generate HTML
- [x] Use tabbed browsing to access the menu
- [x] Use separate files to store menu-related code

Suggestions are welcomed!

### CODE REVIEW

**Comment by bellom** [comment]
> Requesting code review for the project.
> 
> All requirements completed.
> 
> Suggestions are welcomed!

Next time you can always add the requirements/features added 🙂

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+- Javascript
+- Webpack
+
+## Settings
+
+- Run `npm install` to have all the needed packages to run the app.
+
+## Live Demo
+
+[Live Demo Link](https://majovanilla.github.io/restaurant-page
```
Comment (bellom) [inline README.md]: Let make our Repo looks professional by adding a live link.
Kindly watch the video tutorial on how to create a live link 👉  [Link](https://www.loom.com/share/76088842a2d14322bbcf65b884f4d42d);

Comment (majovanilla) [reply README.md]: The live version link is set and ready. I use GitHub pages and the link will work ONCE I merge with master.

Comment (bellom) [reply README.md]: I used `raw.githack` with your repo and everything works fine, but make sure to use `GitHub pages` later on.

**Pull request merged**